layers/+tools/restclient/packages.el

```diff
@@ -35,8 +38,29 @@
       (unless restclient-use-org
         (add-to-list 'auto-mode-alist '("\\.http\\'" . restclient-mode)))
       (spacemacs/set-leader-keys-for-major-mode 'restclient-mode
+        "n" 'restclient-jump-next
```

Comment:
Isn't `SPC g` prefix for git? Looking at the docs I think there's some precedent for `SPC m g n` but that seems little unnecessarily cumbersome when almost the whole major mode prefix is unused. If either the toplevel were more full or the `g` sublevel had more commands it would be a good fit, but as it stands two toplevel commands seems better tradeoff as it is more shallow and takes only one extra key compared to one submenu with only two commands.

*shrug* no strong feelings